azure-pipelines.yml

```
@@ -49,60 +49,44 @@ jobs:
       name: linux
       vmImage: ubuntu-18.04
       matrix:
+        # manylinux 2014
+        pypy_3.7_64:
+          MB_PYTHON_VERSION: "pypy3.7-7.3.4rc1"
```

PyPy 7.3.5 was released.
```suggestion
          MB_PYTHON_VERSION: "pypy3.7-7.3.5"
```